thumberd/thumberd.go

```diff
@@ -481,11 +481,32 @@ func thumbServer(w http.ResponseWriter, r *http.Request, sem chan int) {
 	}
 	defer srcReader.Body.Close()
 
+	fmt.Printf("%#v\n", params)
```

Comment:
is this necessary in prod code? (sorry I'm not very familiar with golang)

Reply:
and I've noticed this is not written by you but rather from the old code

Reply:
Thank you for checking, thumbnail service will run in docker, all stdout output will be redirected to log file, so we feel it is a print to stdout, it will become log actually. 

It is more about container deployment than golang, I think there is argument on how to log in container environment... but let us keep old style at present.

```
stdout_logfile=/data/log/app/thumberd.log
```